custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -2,15 +2,14 @@
 
 window.angular.module('icdsApp').factory('baseControllersService', function() {
     return {
-        BaseController: function ($scope, $routeParams, $location, locationsService, dateHelperService, userLocationId,
-            storageService, haveAccessToAllLocations, haveAccessToFeatures) {
+        BaseController: function ($scope, $routeParams, $location, locationsService, dateHelperService,
+                  navigationService, userLocationId, storageService, haveAccessToAllLocations, haveAccessToFeatures) {
```

Comment:
This is the second major change to `BaseController` - the addition of the `navigationService`. Again, the treatment applied to `underweight-children-report.directive.js` in this PR would later be applied to all "subclasses" of it.